### PR TITLE
改寫建立container方法；新增 gem net-ssh

### PR DESCRIPTION
**Pull request by noracami**
Feature/rewrite add container that exec code

### CODE REVIEW

**Review by zxcvbnm7548 — APPROVED** [review]

**Review by chutienyuan — APPROVED** [review]

**Review by oSeanChen — APPROVED** [review]

**Review by Yojijun — APPROVED** [review]

File: app/views/rooms/invite.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1 class="text-9xl">invite</h1>
+<%= form_with do |form| %>
+  <%= form.email_field :email,
```
Comment (noracami) [inline app/views/rooms/invite.html.erb:3]: 這邊可以幫加個 form.text_field :username

File: app/mailers/room_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RoomMailer < ApplicationMailer
+  def send_invitation_to(user, room)
+    @user = user
+    @room_url = Rails.application.routes.url_helpers.room_uuid_url(uuid: room.uuid, host: 'localhost', port: 3000)
```
Comment (noracami) [inline app/mailers/room_mailer.rb:6]: host 用環境變數

**Review by RexXKao — APPROVED** [review]

File: app/mailers/room_mailer.rb (hunk 2 above)
Comment (noracami) [reply app/mailers/room_mailer.rb:6]: [#08a8569](https://github.com/we-have-TTA/iCoder/pull/70/commits/08a856953bceb375f0709325a3a129509d4a5eb3)

File: app/views/rooms/invite.html.erb (hunk 1 above)
Comment (noracami) [reply app/views/rooms/invite.html.erb:3]: https://github.com/we-have-TTA/iCoder/pull/70/commits/08a856953bceb375f0709325a3a129509d4a5eb3

**Pull request merged**